Automattic/Gravatar-SDK-Android
# QuickEditor - AvatarPicker that load avatars from backend

Closes #223

### Description

This PR implements a composable that, given an email, tries to load the already uploaded avatars. We need to iterate over the loading and error handling.

<img width="250" alt="image" src="https://github.com/user-attachments/assets/1e40d069-f9fd-41d1-adaa-ee497794174e">
<img width="250" alt="image" src="https://github.com/user-attachments/assets/feaf2500-9fc5-49df-826a-b353f984e7db">

### Testing Steps

**Important Note**: The backend is currently returning the wrong format for the `updated date` field. As a workaround, you can apply this patch to comment that field from the Avatar object.
[Gravatar-Android-13-04-48.patch](https://github.com/user-attachments/files/16455284/Gravatar-Android-13-04-48.patch)

1. Run the **demoApp**
2. Navigate to the `Avatar Update` tab
3. Follow the OAuth flow
4. When the OAuth is authorized, you should see the AvatarPicker with your avatars
5. Tap on any of them and verify you can see the toast
